mypy/test/testgraph.py

```diff
@@ -38,7 +39,8 @@ def _make_manager(self):
             ignore_prefix='',
             source_set=None,
             reports=None,
-            options=Options())
+            options=Options(),
+            version_id=__version__)
```

Comment:
I'd add a trailing comma and move the `)` to the next line, to be kind on the next person who adds another parameter here.